k8s/apps/elasticsearch.yml

```
@@ -0,0 +1,47 @@
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: mindjunkies-elasticsearch
+  labels:
+    app: mindjunkies-elasticsearch
+spec:
+  type: ClusterIP
+  ports:
+  - port: 9200
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)